atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/ResilientLockWatchEventCacheTest.java

```diff
@@ -54,6 +56,21 @@ public void before() {
         proxyCache = ResilientLockWatchEventCache.newProxyInstance(defaultCache, fallbackCache, metricsManager);
     }
 
+    @Test
+    public void testCanDelegateIsEnabled() {
+        assertThat(proxyCache.isEnabled()).isTrue();
+
+        RuntimeException runtimeException = new RuntimeException();
+        when(defaultCache.getCommitUpdate(anyLong())).thenThrow(runtimeException);
+        assertThatThrownBy(() -> proxyCache.getCommitUpdate(0L)).hasCause(runtimeException)
+                .isExactlyInstanceOf(TransactionLockWatchFailedException.class);
```

Comment:
do you mind expanding this test with `hasMessage` as there are a number of exceptions under that class (I know I am guilty of this too, but in an effort to make it clearer what we are testing, this should be done).

Reply:
Nope, ResilientLockWatchEventCache has just one from what I can see?

Reply:
Ah yea, I see you're just throwing a generic exception instead of actually causing a legit exception inside LWECI.